ibtk/include/ibtk/FEDataManager.h

```diff
@@ -107,6 +107,9 @@ class FEDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::St
 public:
     /*!
      * Class which enables fast lookup of dofs on a given libMesh <code>elem</code>.
+     *
+     * @note The contents of this cache are invalidated when we regrid and the
```

Comment:
I guess maybe we should say that these are intended to be accessed only via `FEDataManager::getDofCache()`?